translate/packages/translate/translate/index.js

```diff
@@ -1,6 +1,17 @@
 // jshint esversion: 9
 
 const translator = 'https://translate.googleapis.com/translate_a/single';
+let ISO6391;
+async function install(pkgs) {
```

Comment:
This is no longer needed. You can mention your package in packages.json to add an external dependency. Example: https://github.com/nimbella/command-sets/blob/b85860daa9d3d16c0f6b900b88670b78f4fdb392/translate/packages/translate/translate/package.json#L11

Reply:
Hey~ I'm a little confused that How do I test this command in Nimbella? If I remove it it will get `Cannot find module iso-639-1`

Reply:
Here's how you can test it: 
- Push the changes to your branch
- Delete the `translate` command from your Slack workspace using `/nc command_delete translate`
- Install your version of `translate` by running `/nc csm_install https://raw.githubusercontent.com/dengliming/command-sets/fix_message/translate/commands.yaml`.

Alternatively, check out https://github.com/nimbella/commander-cli, it helps you develop and test commands directly from the terminal without having to go Slack.

Reply:
@satyarohith 👍  Thanks a lot.